content/concepts/tokens/nftoken-auctions.md

```diff
@@ -0,0 +1,60 @@
+---
+html: nftoken-auctions.html
+parent: non-fungible-tokens.html
+blurb: You can assign another account to mint NFTs in your stead.
+labels:
+ - Non-fungible Tokens, NFTs
+---
+
+# Running an NFT Auction
+
+There are several ways to run an auction, each with advantages and disadvantages.
+
+## Run the Auction Off the XRPL, Complete the Purchase on XRPL
+
+This flow is listed first as it is the most straightforward. You store your bids in a private database. You take a cut of the winning bid prior to sending the Buyer/Seller the XRPL transaction that completes the purchase. Note that because NFTokenOffer objects can always be canceled by their creator, it is not currently possible to implement a binding offer.
```

Comment:
35 years of leaving contractions out of developer documentation is a hard habit to break, particularly since I don't agree. Our style guide does say we can use common contractions, but Tina will update the guide to avoid some of the more awkward ones.